turbinia/task_manager.py

```diff
@@ -43,6 +43,7 @@
 log = logging.getLogger('turbinia')
 
 PSQ_TASK_TIMEOUT = 604800
```

Comment:
Are these seconds?

I opt to make it clear in the constant name, e.g:
PSQ_TASK_TIMEOUT_SECONDS

Reply:
Yeah, seconds.  I'll update the name.